src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
```

Comment:
LGTM! The only thing I don't understand, why did you moved info about quantity from transaction to fruit. Fruit shoul hold an info about fruit (name, taste, color for example) and transaction is an information about addind the fruit to the storage or buyiing fromstorage and so on.